src/System.IO.Compression/src/System/IO/Compression/Crc32Helper.cs

```diff
@@ -495,5 +529,11 @@ static public uint UpdateCrc32(uint crc32, byte[] buffer, int offset, int length
             crc32 ^= 0xFFFFFFFFU;
             return crc32;
         }
+
+        [SecurityCritical]
+        static private uint crc32_zlib(uint crc32, byte[] buffer, int offset, int length)
```

Comment:
Nit: I realize there's already some inconsistency in this file, but can you change the methods to be ```visibility static``` rather than ```static visibility```?